module-system/module-implementations/sov-evm/src/hooks.rs

```diff
@@ -9,6 +11,48 @@ impl<C: sov_modules_api::Context> Evm<C> {
         _da_root_hash: [u8; 32],
         _working_set: &mut WorkingSet<C::Storage>,
     ) {
+        let block_number: u64 = self.head_number.get(_working_set).unwrap();
+        let parent_block: reth_rpc_types::Block = self
+            .blocks
+            .get(&block_number, &mut _working_set.accessory_state())
```

Comment:
When we process block `0` `self.blocks` will be empty?

Reply:
I need to initialize block 0 in genesis